src/app/content/utils/seoUtils.ts

```diff
@@ -9,20 +12,182 @@ import {
   splitTitleParts,
 } from './archiveTreeUtils';
 
-const getParentPrefix = (node: LinkedArchiveTreeNode | undefined): string => {
+type PageTypes = 'page' | 'appendix' | 'answer-key' | 'eoc-sub-page' | 'eoc-page' | 'eob-page';
+
+interface DescriptionTemplateValues {
+  parentTitle: string;
+  parentType: 'chapter' | 'book' | 'other';
+  pageTitle: string;
+  bookTitle: string;
+  chapterTitle: string;
+}
+
+const domParser = new DOMParser();
+
+export const getParentPrefix = (node: LinkedArchiveTreeNode | undefined, includeTitle: boolean = false): string => {
   if (!node) {
     return '';
   }
 
   if (archiveTreeSectionIsChapter(node)) {
-    const number = getArchiveTreeSectionNumber(node);
-    return `Ch. ${number} `;
+    const number = getArchiveTreeSectionNumber(node).replace('Chapter', '').trim();
```

Comment:
where did you see something with the word `chapter` in the number?

Reply:
It only happened when I ran the metadata dump script we wrote that pulls books from config.books.json

Reply:
do you know what book it was in?

Reply:
All of them. When I log `getArchiveTreeSectionNumber(node)`, which pulls the text content of the from the first `.os-number` element, on my local env it contains only a number (eg "2"). When I log it via the script it logs "Chapter 2"

Reply:
ah, i was wondering how you could not be getting the logic that cleans up the tree, but apparently its not set up to run until you combine the archive book with the cms data. if you look in ` src/app/content/utils.ts` there is some logic there that cleans the tree titles. i think all the logic in the app is going to expect that cleaning logic to have been run.

i think that if your logic were to run on the actual page, the place where the data comes from would have been cleaned already, but in your script you're loading it from the archive loader directly and its not going through the same logic

Reply:
do you think we need that cleanup logic here or will it suffice to just tidy the `parentPrefix` string within the script file? I'm struggling to find which util function I would need to apply

Reply:
ah but then I have the same issue when running tests

Reply:
honestly you might want to change your meta util to accept the combined book and then run everything including your test data through `formatBookData` and that'll handle it for you. you'll see other tests using `formatBookData` as well